### ISSUE TITLE
IgnoreForCreate attribute

### ISSUE THREAD

**Issue by fjrodriguezcom** [praeclarum/sqlite-net#72]
Would be great if I could have an entity like this:

class Client
{
     string IdClient;

```
 [IgnoreForCreate]
 string otherDate;
```

}

So I could do a select with inner join and fill otherDate but not create it into database. It's like a ignore but not for select.

**Comment by rsnitsch** [comment]
I agree that this would be very helpful. I have created a similar feature request here (but I was not aware that sqlite-net is responsible for these attributes):
https://bitbucket.org/twincoders/sqlite-net-extensions/issues/90/improve-ignore-attribute-behavior-when

This would allow me to write queries like this:
```
SELECT s.*, p.JoinedPhotoData AS JoinedPhotoData
    FROM Student AS s
    JOIN Photo AS p ON p.Id == s.PhotoId
```

... where `JoinedPhotoData` would be a property of the Student object with the IgnoreForCreate attribute.
